_layouts/home.html

```diff
@@ -113,30 +142,27 @@ <h3 class="card-title">Code of Conduct</h3>
         </div>
       </div>
     </div>
-    {% endcomment %}
   </div>
 </section>
 
 {% assign visible_posts = site.posts | where_exp: 'item', 'item != false' %}
 
 {% if visible_posts.size > 0 %}
-<section class="news-section section-pad theme-brand-color4 back">
+<section class="news-section section-pad">
   <div class="row column v-pad-bottom">
     <h2 class="text-center">Latest News</h2>
   </div>
 
-  <div class="news row">
+  <div class="news row column">
     {% for post in visible_posts limit:12 %}
-      {% unless post.hidden %}
-      <article class="news-title medium-4 column">
-        <div class="card {% if post.category == "Sponsorship" %}card-theme-brand-color3{% else %}card-theme-brand-color2{% endif %}">
-          <div class="card-section">
-            <span class="post-meta">{{ post.date | date: "%d %b %Y" }}</span>
-            <h3>
-              <a href="{{ post.url | prepend: site.baseurl }}">{{ post.title | escape }}</a>
-            </h3>
-          </div>
-        </div>
+      {% unless post.hidden or post.category == 'talk' or post.category == 'tutorial' %}
```

Comment:
This and below – copy of `post-list.html`, so we get the same results on the news page and here.